cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```
@@ -513,6 +518,7 @@ objects:
           - --cloud-region-config=/configs/cloud-resources/cloud-regions.yaml
           - --cloud-region-constraints-config=/configs/cloud-resource-constraints/cloud-region-constraints.yaml
           - --azure-auth-config-path=/secrets/azure-credentials/azure-auth-config
+          - --azure_fpa_client_id=${AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID}
```

The `azure_fpa_client_id` is only used for the makefile. This should be: 

```suggestion
          - --azure-first-party-application-client-id=${AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID}
```